src/main/java/com/example/provider/VehicleServlet.java

```diff
@@ -45,6 +48,12 @@ public class VehicleServlet extends HttpServlet {
   private static final String SUPPORTED_POST_LINK = "/new";
   private static final String VEHICLE_ID_FIELD = "vehicleId";
   private static final String BACK_TO_BACK_ENABLED_FIELD = "backToBackEnabled";
+  private static final String SUPPORTED_TRIP_TYPES_FIELD = "supportedTripTypes";
+  private static final String MAXIMUM_CAPACITY_FIELD = "maximumCapacity";
+
+  private static final int MAXIMUM_CAPACITY_DEFAULT = 5;
```

Comment:
minor nit: Should we use 4 for the default max capacity here to stay consistent with the current max capacity?

Reply:
Good point, updated